test/tests.test.ts

```diff
@@ -31,6 +31,14 @@ const fetchTextContent = async (
   return content
 }
 
+const fetchEtagHeader = async (
```

Comment:
What is the reason to use ETag? I think using `res.arrayBuffer()` is more simple and consistent.

Reply:
how do you suggest to compare them?

Reply:
You could use `toStrictEqual`.
https://stackblitz.com/edit/vitest-dev-vitest-r7tfbp?file=test%2Fbasic.test.ts&initialPath=__vitest__